### DISCUSSION TITLE
Receive webhook when topic changes | GitHub Enterprise v3.2.1

### DISCUSSION THREAD

**Discussion by karamws** [community/community#39936] [Enterprise]
### Select Topic Area

Question

### Body

I do not receive a webhook when I change the topic list.

- Metadata access is enabled 
- Repository is checked 
- current GitHub Enterprise server version is 3.2.1

**Comment by ossanna16** [comment]
Hi there @karamws and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
